pkgs/applications/misc/far2l/default.nix

```
@@ -79,7 +68,7 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   meta = with stdenv.lib; {
-    description = "An orthodox file manager";
+    description = "Linux port of FAR Manager v2 (http://farmanager.com/)";
```

```suggestion
    description = "Linux port of FAR Manager v2";
```

Homepage is for links.

---

Homepage is the upstream project, which is a Linux port of the original product. Might not tell people much.
This one link is to the original product the Linux port is trying to mimic, which should be more widely known.

---

Not in the description. Link do not belong there.

---

Okay.